formvalidator/src/main/java/dev/poteto/formvalidator/rules/ValidateCurrency.kt

```diff
@@ -0,0 +1,44 @@
+package dev.poteto.formvalidator.rules
+
+import dev.poteto.formvalidator.DataType
+import dev.poteto.formvalidator.contracts.PredefinedValidationRule
+import dev.poteto.formvalidator.contracts.ValidationMessages
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+/**
+ * Class for validating email
```

Comment:
Use the appropriate description for the certain class

Reply:
done